.pre-commit-config.yaml

```diff
@@ -70,6 +70,10 @@ repos:
       - id: rst-inline-touching-normal
         types: [text]  # overwrite types: [rst]
         types_or: [python, rst]
+-   repo: https://github.com/sphinx-contrib/sphinx-lint
+    rev: main
```

Comment:
can you put a tag (or a commit)? `main` isn't supported

Reply:
https://pre-commit.com/#using-the-latest-version-for-a-repository

Reply:
I used a tag instead.